elcid/views.py

```diff
@@ -100,3 +104,105 @@ def get_context_data(self, *args, **kwargs):
             pass
 
         return ctd
+
+
+def ward_sort_key(ward):
+    """"
+    Example wards are
+            "8 West",
+            "PITU",
+            "ICU 4 East",
+            "9 East",
+            "8 South",
+            "9 North",
+            "ICU 4 West",
+            "12 West",
+
+    We want to order them first by number
+    then with the wards that are strings.
+    e.g. ICU
+    """
+
+    start = ward[:2].strip()
+    rest = ward[2:]
+    return (len(start), start, rest)
+
+
+class RenalHandover(LoginRequiredMixin, TemplateView):
+    template_name = "elcid/renal_handover.html"
+
+    def get_unit_ward(self, location):
+        if location.ward and location.bed:
+            return "{}/{}".format(
+                location.ward, location.bed
+            )
+
+    def get_context_data(self, *args, **kwargs):
+        """
+        An ordered dictionary of ward to patient
+
+        patient here is...
+        their name
+        their hospital number
+        their unit/ward
```

Comment:
This is actually 'their ward/bed' - stealing the terminology makes the docstring confusing...